sdk/compatability/src/lib.rs

```diff
@@ -0,0 +1,694 @@
+        ),
+        transaction_builder::encode_create_validator_operator_account_script(
+            0,
+            AccountAddress::ZERO,
+            Vec::new(),
+            Vec::new(),
+        ),
+        transaction_builder::encode_remove_validator_and_reconfigure_script(
+            0,
+            Vec::new(),
+            AccountAddress::ZERO,
+        ),
+        transaction_builder::encode_rotate_authentication_key_with_nonce_admin_script(
+            0,
+            Vec::new(),
+        ),
+        transaction_builder::encode_rotate_authentication_key_with_nonce_script(0, Vec::new()),
+        transaction_builder::encode_set_validator_operator_with_nonce_admin_script(
+            0,
+            Vec::new(),
+            AccountAddress::ZERO,
+        ),
+        transaction_builder::encode_update_diem_version_script(0, 0),
+        // TreasuryCompliance Scripts
+        transaction_builder::encode_burn_script(Currency::XDX.type_tag(), 0, AccountAddress::ZERO),
+        transaction_builder::encode_burn_txn_fees_script(Currency::XDX.type_tag()),
+        transaction_builder::encode_cancel_burn_script(
+            Currency::XDX.type_tag(),
+            AccountAddress::ZERO,
+        ),
+        transaction_builder::encode_create_designated_dealer_script(
+            Currency::XDX.type_tag(),
+            0,
+            AccountAddress::ZERO,
+            Vec::new(),
+            Vec::new(),
+            false,
+        ),
+        transaction_builder::encode_create_parent_vasp_account_script(
+            Currency::XDX.type_tag(),
+            0,
+            AccountAddress::ZERO,
+            Vec::new(),
+            Vec::new(),
+            false,
+        ),
+        transaction_builder::encode_freeze_account_script(0, AccountAddress::ZERO),
+        transaction_builder::encode_preburn_script(Currency::XDX.type_tag(), 0),
+        transaction_builder::encode_tiered_mint_script(
+            Currency::XDX.type_tag(),
+            0,
+            AccountAddress::ZERO,
+            0,
+            0,
+        ),
+        transaction_builder::encode_unfreeze_account_script(0, AccountAddress::ZERO),
+        transaction_builder::encode_update_dual_attestation_limit_script(0, 0),
+        transaction_builder::encode_update_exchange_rate_script(Currency::XDX.type_tag(), 0, 0, 0),
+        transaction_builder::encode_update_minting_ability_script(Currency::XDX.type_tag(), false),
+        // Validator Operator Scripts
+        transaction_builder::encode_register_validator_config_script(
+            AccountAddress::ZERO,
+            Vec::new(),
+            Vec::new(),
+            Vec::new(),
+        ),
+        transaction_builder::encode_set_validator_config_and_reconfigure_script(
+            AccountAddress::ZERO,
+            Vec::new(),
+            Vec::new(),
+            Vec::new(),
+        ),
+        transaction_builder::encode_set_validator_operator_script(Vec::new(), AccountAddress::ZERO),
+    ]
+    .to_vec();
```

Comment:
So this was the first test I actually wrote because its important to know that the older txns we plan to continue to support are advertised as such, but it turns out that when spinning up local testnets this list isn't populated (so this test is actually marked as failing atm). I need to get some input from @tzakian if this is a reasonable test or if I should just scrape it.

Reply:
I would love to scrap this test, but I think this is a reasonable test to keep around if for no other reason but to force us to confront what we want to do with it when we remove scripts. 

I would remove the `burn` and `cancel_burn` scripts as those will no longer be supported in V2 (and it would be nice to remove them). Additionally, I'm not sure how useful it will be to keep the `*_admin_script` scripts in here as admin scripts of any type will be allowed (so removing them from this list would be a non-breaking change).